### ISSUE TITLE
IBUS issue

### ISSUE THREAD

**Issue by general03** [timothycrosley/streamdeck-ui#336]
When I launch `streamdeck` I get 
```
qt.dbus.integration: Could not connect "org.freedesktop.IBus" to globalEngineChanged(QString)
No Stream Deck(s) detected. Attach a Stream Deck.
```

**To Reproduce**
I'm on **Ubuntu 22.04.2 LTS** and use **Python 3.10.6**, when I launch `streamdeck`  I see the UI but the StreamDeck is not detected

**Expected behavior**
Do not have this issue on the console and see on UI the device under the list 

**Screenshots**
![image](https://github.com/timothycrosley/streamdeck-ui/assets/2324694/ace15269-5e13-4968-b673-696181d22696)

**Operating system (please complete the following information)**
 - Ubuntu 22.04.2 LTS on Budgie
 - Python 3.10.6

**Stream Deck Version**
 - Stream Deck + : https://www.ldlc.com/fiche/PB00547463.html?offerId=AR202211170033

**Comment by general03** [comment]
If I launch `pkill ibus-x11` the IBUS issue disappear,

But I can't see my Stream Deck

**Comment by general03** [comment]
If someone can help me on the device connection, I can contribute

**Comment by bcdonadio** [comment]
This hardware version is not supported by the underlying library. You might have luck by adding your USB_PID_VID in the product lists here https://github.com/abcminiuser/python-elgato-streamdeck/tree/master/src/StreamDeck and copying the code from the closest sibling, with the differences that you know for a fact, creating a new file here: https://github.com/abcminiuser/python-elgato-streamdeck/tree/master/src/StreamDeck/Devices

**Comment by general03** [comment]
Yes I already try this without success, I never develop library with device communication so I do not know how to start

**Comment by 4s3ti** [comment]
hi, this project seems to be stale and a new fork as been made with the intention of replacing this one.

if this still an issue please reopen it at: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui

**Comment by general03** [comment]
Yes I always have this issue, I reported it on new repo https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/issues/10

Thanks
